openrouteservice/src/main/java/heigit/ors/isochrones/builders/concaveballs/ConcaveBallsIsochroneMapBuilder.java

```diff
@@ -32,10 +32,9 @@
 import heigit.ors.isochrones.builders.AbstractIsochroneMapBuilder;
```

Comment:
Do a quick cleanup of the imports as it seems some are not needed anymore

Reply:
done